data/json/items/book/traps.json

```diff
@@ -52,9 +52,8 @@
   {
     "id": "textbook_traps",
     "type": "BOOK",
-    "name": "The Compleat Trapper",
-    "name_plural": "The Compleat Trapper",
-    "//": "The spelling is intentionally archaic, as this form is commonly used in the titles of books.",
```

Comment:
Got it. Plus, it's such a little-used variant that I'd expect players would likely think it's just a misspelling.